### DISCUSSION TITLE
Custom theme colors

### DISCUSSION THREAD

**Discussion by lcambourian** [colinwilson/lotusdocs#172] [Q&A]
Hi! I see there are a few options for the theme color, but can we adjust them or create a new one? I tried and failed unless I changed a file from the cache... I'd like to use the exact colors of my graphic charter.

And same question for all styles, for example if I want to adjust spacing, font size, etc.

Thanks!

**Comment by lcambourian** [comment]
Sorry I found my answer: the changes I made in the scss files were not considered because my hugo.toml file was importing the theme as a module from github.
![image](https://github.com/colinwilson/lotusdocs/assets/90646776/f59b0fd2-0f69-4ed1-8801-e336f6d418e7)

I just had to add a line **theme = "lotusdocs"** (with the lotusdocs theme being in the themes folder) and disable the module import.
![image](https://github.com/colinwilson/lotusdocs/assets/90646776/d6868ebc-5394-4010-9728-37a0595f3b09)

**Comment by fenglisch** [comment]
You can also do this while importing the theme from Git:
- Create a file `assets/docs/scss/custom/colors/_mycolor.scss` with the same content as for example the [_red.scss](https://github.com/colinwilson/lotusdocs/blob/release/assets/docs/scss/custom/colors/_red.scss), but with different color values.
-  Add this to your toml:

```
[module]
    [[module.mounts]]
        source = "assets"
        target = "assets"

[params.docs]
    themeColor = "mycolor" 
```
